packages/dd-trace/src/iitm.js

```diff
@@ -2,8 +2,24 @@
 
 const semver = require('semver')
 const logger = require('./log')
+const shimmer = require('../../datadog-shimmer')
+const dc = require('diagnostics_channel')
 
 if (semver.satisfies(process.versions.node, '>=14.13.1')) {
+  const moduleLoadStartChannel = dc.channel('dd-trace:moduleLoadStart')
+  const iitm = require('import-in-the-middle/lib/register.js')
+  shimmer.wrap(iitm, 'register', register => {
```

Comment:
Can we avoid patching iitm? We own the module so we can make changes there if needed.

Reply:
I didn't know that we are the owners. I'm going to do another PR in the module project.

Reply:
Ok, I have opened a PR to iitm: https://github.com/DataDog/import-in-the-middle/pull/19
With this PR, I'm going to be able to change `shimmer.wrap` by `iitm.addHook`